### DISCUSSION TITLE
Is it possible to (hyper)link directly to a Trilium note from an external app?

### DISCUSSION THREAD

**Discussion by myii** [TriliumNext/Trilium#1407] [Q&A] [answered]
I've got some node-based diagrams (using a separate app), which can allow adding a hyperlink that can be opened for each node.  I'd like to be able to provide hyperlinks to specific notes in Trilium for various nodes on the diagram.  Is this possible?  I'm supposing it could be based around something like `http://localhost:37840/...`.

**Comment by zadam** [comment, marked as answer]
This is currently possible only for server version.

For desktop no, at least not yet. It would have to work via a registered custom protocol, something like `trilium://noteid` which would then trigger trilium app.

**Reply by myii** [reply]
Appreciate the response.
